core/fetcher/fetcher.go

```diff
@@ -146,6 +147,20 @@ func (f *Fetcher) fetchAttesterData(ctx context.Context, slot int64, defSet core
 		}
 
 		resp[pubkey] = attData
+
+		// Store Attestation data root excluding committee index.
+		data := *eth2AttData
```

Comment:
@corverroos using value of `eth2AttData` now rather than modifying pointer.

Reply:
what about `attData.Clone()`?